scripts/run_failpoint_tests.py

```
@@ -0,0 +1,115 @@
+        run([str(failpoint_ctl), "enable", "."], cwd=repo_root, env=env)
+        for pkg, tests in sorted(tests_by_pkg.items()):
+            pattern = "^(%s)$" % "|".join(sorted(tests))
+            run(
+                ["go", "test", "-v", "-tags", "failpoint", pkg, "-run", pattern],
+                cwd=repo_root,
+                env=env,
+            )
+    except subprocess.CalledProcessError as exc:
+        exit_code = exc.returncode or 1
+    finally:
+        subprocess.run([str(failpoint_ctl), "disable", "."], cwd=repo_root, check=False, env=env)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)